pkgs/tools/misc/pandoc-acro/default.nix

```
@@ -0,0 +1,60 @@
+{ buildPythonApplication
+, fetchPypi
+, pandocfilters
+, panflute
+, lib
+, pandoc
+, pandoc-acro
+, texlive
+, runCommand
+}:
+
+let
+  pname = "pandoc-acro";
+  version = "0.10.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-JMfSQXX+BCGdFQYPFB+r08WRnhT3aXfnBNINROxCUA0=";
+  };
+in
+buildPythonApplication rec {
```

```suggestion
buildPythonApplication {
```

If I'm reading it right, you don't need `rec`, because you're already using an explicit `let`.